### PR TITLE
Fix nGQL tutorial's comment and table

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: docs/nGQL-tutorial.md [hunk 1]
```diff
@@ -311,9 +307,10 @@ you to follow these syntax norms:
...
-  | Graph entity  | Recommended style | Example |
-  |:-: | :-: | :-: |:-: |
-  |Key words | Upper case   | SHOW SPACES     |
-  |Vertex tags | Upper camel case, beginning with an upper-case character   | ManageTeam   |
-  |Edges | Upper snake case, beginning with an upper-case character   | Play_for   |
-  |Property names | Lower camel case, beginning with a lower-case character   | inService   | --> -->
+| Graph entity  | Recommended style                                          | Example     |
+|:-:            | :-:                                                        | :-:         |
+|Key words      | Upper case                                                 | SHOW SPACES |
+|Vertex tags    | Upper camel case, beginning with an upper-case character   | ManageTeam  |
```
Comment (whitewum) [inline docs/nGQL-tutorial.md]: It looks like the unittest, and our examples are different.

How about we comment out the style section? until we have agreements.

Comment (darionyaphet) [reply docs/nGQL-tutorial.md]: Correctly! We should keep the style unify.

**Comment by jude-zhu** [comment]
Jenkins go

**Comment by darionyaphet** [comment]
Jenkins go

**Pull request merged**